dbms/src/WindowFunctions/tests/gtest_last_value.cpp

```diff
@@ -134,32 +132,66 @@ class LastValue : public DB::tests::ExecutorTest
 TEST_F(LastValue, lastValue)
 try
 {
-    // TODO test with bounded_type_frame
-    MockWindowFrame unbounded_type_frame{
-        tipb::WindowFrameType::Rows,
-        std::make_tuple(tipb::WindowBoundType::Preceding, true, 0),
-        std::make_tuple(tipb::WindowBoundType::Following, true, 0)};
-
-    executeFunctionAndAssert(
-        toVec<String>({"1", "5", "5", "5", "5", "10", "10", "10", "10", "10", "13", "13", "13"}),
-        LastValue(value_col),
-        {toVec<Int64>(/*partition*/ {0, 1, 1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 3}),
-         toVec<Int64>(/*order*/ {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12}),
-         toVec<String>(/*value*/ {"1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13"})},
-        unbounded_type_frame);
-
-    executeFunctionAndAssert(
-        toNullableVec<String>({{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}}),
-        LastValue(value_col),
-        {toNullableVec<Int64>(/*partition*/ {0, 1, 1, 1, 1, 2, 2, 2, 2, 2, 3, 3, 3}),
-         toNullableVec<Int64>(/*order*/ {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12}),
-         toNullableVec<String>(/*value*/ {{}, "2", "3", "4", {}, "6", "7", "8", "9", {}, "11", "12", {}})},
-        unbounded_type_frame);
+    {
+        // frame type: unbounded
```

Comment:
and add some tests in `tests/fullstack/`

Reply:
> and add some tests in `tests/fullstack/`

okk, I will add some ft